apps/condo/domains/ticket/utils/handlers.js

```diff
@@ -195,34 +211,44 @@ const handleTicketEvents = async (requestData) => {
     }
 }
 
-const sendDifferentKindsOfNotifications = async (requestData) => {
+const handleTicketCommentEvents = async (requestData) => {
```

Comment:
it's too abstract! use names that you don't need to know the code/task context to understand whats going on inside the function

Reply:
Please don’t do abstract function naming. At the moment you use it for one task. You will need to create another one function for another task. 

It’s a bad practice to use abstract naming. It doesn’t help you team to understand ways going on without reading the code.

Reply:
fixed